service-discovery/src/main/resources/rp-tooling.conf

```diff
@@ -6,8 +6,10 @@ akka {
     dns {
       resolver = async-dns
       async-dns {
+        # FIXME: temporarily disable this until we sort out the issue around shaded Akka DNS dependency
         # This has to be set to the name of the shaded class
-        provider-object = "com.lightbend.rp.internal.ru.smslv.akka.dns.AsyncDnsProvider"
+        # provider-object = "com.lightbend.rp.internal.ru.smslv.akka.dns.AsyncDnsProvider"
+        provider-object = "ru.smslv.akka.dns.AsyncDnsProvider"
```

Comment:
Do we really need it right now?

Reply:
Yes, we do need it for looking up other service through DNS SRV.

Reply:
That's what I'm asking -- is SRV needed right away? Things work with just A on kubernetes, please explain more about how urgent support o SRV is

Reply:
It's required right away, yes. Otherwise Lagom service to another Lagom service communication won't work, i.e. like in Chirper or Online Auction example.

Reply:
So it's about the port, right? I get it if so. Trying to get the real why-it's-needed 😉